app/controllers/projects/blob_controller.rb

```diff
@@ -44,11 +50,16 @@ def update
     result = Files::UpdateService.new(@project, current_user, @commit_params).execute
 
     if result[:status] == :success
-      flash[:notice] = "Your changes have been successfully committed"
```

Comment:
I think it should remain at above, so even redirect_to or render not working, the user still know the result of post or put.